src/context/AuthContext/index.tsx

```diff
@@ -47,6 +48,7 @@ export const AuthProvider: React.FC<AuthProviderProps> = ({ children }) => {
 
   const defaultChainId = chains[0].id;
   const chainId = chain?.id || defaultChainId;
+  Sentry.setTag('chainId', chainId);
```

Comment:
This will be triggered every time the component renders, not just when `chainId` changes.
A quick solution for now would be to wrap the call in an effect and set `chainId` as its dependency.

However, another approach could be to make a separate error package, which would export an `ErrorProvider` that renders the Sentry provider as well as the effect for automatically updating the chainId in Sentry (I think we should [add the account address](https://docs.sentry.io/platforms/javascript/session-replay/configuration/) as well). We'll then be able to move all the functions and hooks we currently have in the error folder into this error package and keep everything related to errors in it.

Let me know your thoughts though, since that's more work.